detekt-api/src/test/kotlin/io/gitlab/arturbosch/detekt/api/ConfigPropertySpec.kt

```diff
@@ -8,7 +8,8 @@ import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
 import java.util.concurrent.atomic.AtomicInteger
 
-class ConfigPropertySpec {
+class
+ConfigPropertySpec {
```

Comment:
I suppose that this is an error

```suggestion
class ConfigPropertySpec {
```